crates/net/network/src/transactions/fetcher.rs

```diff
@@ -1111,7 +1112,7 @@ impl VerifyPooledTransactionsResponse for UnverifiedPooledTransactions {
     fn verify(
         self,
         requested_hashes: &[TxHash],
-        peer_id: &PeerId,
+        _peer_id: &PeerId,
```

Comment:
didn't know that variables `_var` could still be used, nice! https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=383517bfe2a99eb05c474d2d1ba6f684